tensorflow_lite_support/ios/task/vision/BUILD

```diff
@@ -24,6 +24,7 @@ objc_library(
         "//tensorflow_lite_support/ios/task/processor:TFLClassificationResultHelpers",
         "//tensorflow_lite_support/ios/task/vision/utils:GMLImageUtils",
         "//tensorflow_lite_support/odml/ios/image:MLImage",
+        "@org_tensorflow//tensorflow/lite/experimental/acceleration/configuration:coreml_plugin",
```

Comment:
I have linked in the cormel_plugin target here. I am still getting the following error:
"Could not create CoreML plugin. Have you linked in the CoreML_plugin target?"